pkg/webhooks/clusterqueue_webhook_test.go

```diff
@@ -369,3 +371,66 @@ func TestValidateClusterQueueUpdate(t *testing.T) {
 		})
 	}
 }
+
+func makeFlavor(name string) kueue.FlavorQuotas {
+	return kueue.FlavorQuotas{
+		Name: kueue.ResourceFlavorReference(name),
+		Resources: []kueue.ResourceQuota{{
+			Name:         corev1.ResourceCPU,
+			NominalQuota: resource.MustParse("1"),
+		}},
+	}
+}
+
+func TestValidateTotalFlavors(t *testing.T) {
+	// Construct >256 flavors
+	var flavors []kueue.FlavorQuotas
+	for i := 0; i < 257; i++ {
```

Comment:
```suggestion
	for i := range 257 {
```
I think, to make linter happy